src/_sass/header.scss

```diff
@@ -1,13 +1,11 @@
 .siteHeader2 {
-  width: 100%;
   display: flex;
-  margin-right: auto;
-  margin-left: auto;
+  margin-right: $spacing-unit;
+  margin-left: $spacing-unit;
 
   @include mobileAndAbove {
-    max-width: $tilt2-header-width;
-    padding-top: $spacing-unit * 2;
-    padding-bottom: $spacing-unit;
+    padding-top: $spacing-unit * 1;
```

Comment:
These changes for the header also apply to the home page and blog, which I don't think is intended. 

I started work already in a separate branch which has a separate header for Docs vs homepage / blog. You can see that I made a separate [header-docs.scss](https://github.com/tilt-dev/tilt.build/blob/hyu/slim-header/src/_sass/header-docs.scss) that corresponds to the separate docs-header.html.

Maybe we can pair and integrate some of my changes with your branch!